src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbolBase.cs

```diff
@@ -98,7 +100,7 @@ public sealed override bool IsVararg
             }
         }
 
-        public sealed override bool IsImplicitlyDeclared
```

Comment:
>public sealed override bool IsImplicitlyDeclared [](start = 8, length = 48)

It looks like this change is unnecessary, given the comment in SynthesizedRecordConstructor.cs.